templates/settings.py.erb

```diff
@@ -72,3 +72,6 @@ LOGGING = {
         }
     }
 }
+
+# pulp_deb plugin default behaviour:
+FORCE_IGNORE_MISSING_PACKAGE_INDICES = True
```

Comment:
This is only relevant for pulp_deb, and as such it must be in a fragment.

In practice it means https://github.com/theforeman/puppet-pulpcore/blob/master/manifests/plugin/deb.pp must ensure the content. See  https://github.com/theforeman/puppet-pulpcore/commit/5858c93e91a29f910aa9eab0e0fb6bd34a69e071 for a similar change.

If you add it, I'd also consider adding a class parameter to change the value.

Reply:
Thanks for the pointers!